views/scored_word.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= @word %></td>
+  <td><%= @score %></td>
+  <td><% if @word %>
+        <% @word.split("").each do |letter| %>
+          <%= "#{letter}: #{Scrabble.score_letter(letter)}" %>
```

Comment:
So we could split the word into its letters and score them in the scrabble.rb file instead, then just calling that method? That sounds way cleaner.